model/model.md

```diff
@@ -3,9 +3,17 @@
 ## 2 Dataspace Information Model
 
 The following sections outline the Dataspace Information Model, which form the foundation of this specification.
+Some aspects of this section describe additional concepts of Dataspaces and provide context for the Dataspace Protocol,
+those are considered as not-normative. Further information on the functional requirements of a Dataspace can be found in
+the [IDSA Rulebook](https://docs.internationaldataspaces.org/idsa-rulebook).
```

Comment:
I don't think these specifications should reference the Rule Book. It should only be the other way around (Rule Book -> DSP)

Reply:
I think we should do it in both directions. When we keep the description and definition of the data space authority and the registry out of this document, then we should at least leave a hint on a place that describes it. I think the Rulebook is a good place, but is there anything else we can refer to instead?

Reply:
I think we should only reference other documents from the specifications if they are associated with a normative requirement. We should not have a normative requirement on the Rulebook or any other document regarding a dataspace authority since it is only a conceptual holder and is not required for interoperability.  

Having a reference to the Rulebook that implies any type of requirement will also lead to practical complications when this work is submitted to a specification group.

Reply:
Adding a for example to the rule book mention